cmd/nerdctl/container_kill_linux_test.go

```diff
@@ -18,12 +18,12 @@ package main
 
 import (
 	"fmt"
-	"regexp"
 	"strings"
 	"testing"
 
 	"github.com/containerd/nerdctl/v2/pkg/rootlessutil"
 	"github.com/containerd/nerdctl/v2/pkg/testutil"
+	iptablesUtil "github.com/containerd/nerdctl/v2/pkg/testutil/iptables"
```

Comment:
```suggestion
	iptablesutil "github.com/containerd/nerdctl/v2/pkg/testutil/iptables"
```

Or just call it "iptables"

Reply:
unfortunately we already have `iptables` used by the package below: `"github.com/coreos/go-iptables/iptables"`.
That's why I was using a different name.

Reply:
```suggestion
	iptablesutil "github.com/containerd/nerdctl/v2/pkg/testutil/iptables"
```